src/tye/UndeployHost.cs

```diff
@@ -17,16 +17,12 @@ namespace Microsoft.Tye
 {
     public static class UndeployHost
     {
-        public static async Task UndeployAsync(IConsole console, FileInfo path, Verbosity verbosity, string @namespace, bool interactive, bool whatIf, string[] tags)
+        public static async Task UndeployAsync(OutputContext output, FileInfo path, string @namespace, bool interactive, bool whatIf, string? framework = null, ApplicationFactoryFilter? filter = null)
         {
             var watch = System.Diagnostics.Stopwatch.StartNew();
 
-            var output = new OutputContext(console, verbosity);
+            var application = await ApplicationFactory.CreateAsync(output, path, framework, filter);
```

Comment:
On a second look, it looks like we don't need to add the framework option to Undeploy at all. The other commands run a build/publish step which will need a framework to be specified. However for undeploy, only the applicationname is resolved from the application. We can just pass null for the framework here.

Reply:
I suppose I also remove this from the Signature ?
```csharp
public static async Task UndeployAsync(OutputContext output, FileInfo path, string @namespace, bool interactive, bool whatIf, ApplicationFactoryFilter? filter = null)
```

Reply:
Yup the change looks good.

Reply:
done